developing_packages/editable_packages.rst

```diff
@@ -21,16 +21,14 @@ get the changes ready for a consumer library, it is needed the :command:`conan c
 actually trigger a build to generate the binaries in the cache or to run :command:`conan export-pkg`
 to copy locally built artifacts into the conan cache and make them available to consumers.
 
-What about if you just can tell Conan where to find the headers and the artifacts ready for
-consumption in your local working directory? No need to package, just tell Conan to use those
-artifacts you have just generated with your IDE, sounds good? This is what the feature
-*editable packages* will do for you.
+With the editable packages, you can tell Conan where to find the headers and the artifacts ready for
+consumption in your local working directory. There is no need to package.
 
 Let's see this feature over an example where a developer is creating a ``CoolApp`` but at the same
-time they wants to work on ``cool/version@user/dev`` library which is tightly coupled to the app.
```

Comment:
"Singular ``they``" is used to omit the gender when referencing a 3rd person, see https://dictionary.cambridge.org/dictionary/english/they. So I think this ok